ENH: Final PR for Python bindings for Seldon (GSOC'24) (#2)
seldon-code/pyseldonlib: docs/install.rst

Reviewer: Use `` `` for code blocks (inline) instead of ``

Installation
=============

Python Package Index (PYPI)
---------------------------

> You can install the package from PYPI using `pip`:

Reviewer: ```suggestion
You can install the package from PYPI using ``pip``:
```

.. code-block:: bash

    $ pip install pyseldon

From Source
--------------

Create and Activate the Environment
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Create a new `micromamba` environment and activate it:

Reviewer: ```suggestion
Create a new ``micromamba`` environment and activate it:
```

.. code-block:: bash

  $ micromamba create -f environment.yml

  $ micromamba activate pyseldonenv

> If you get problem with micromamba can look at the installation documentation `here <https://mamba.readthedocs.io/en/latest/installation/micromamba-installation.html>`_

Reviewer: ```suggestion
For ``micromamba`` related installation issues, consult the upstream documentation `here <https://mamba.readthedocs.io/en/latest/installation/micromamba-installation.html>`_
```

Build
~~~~~
Set up the build directory with `meson`:

.. code-block:: bash

   $ meson setup build

Install Python package
~~~~~~~~~~~~~~~~~~~~~~~

> Use `pip` to install the package:

Reviewer: ```suggestion
Use ``pip`` to install the package:
```
